### DISCUSSION TITLE
What are the limitations of pockethost.io?

### DISCUSSION THREAD

**Discussion by aonghas** [pockethost/pockethost#175] [Q&A]
Hi there,

Just wondering what limitations there are for the free version of pockethost? Storage? Speed? etc? How can I check usage?

**Comment by benallfree** [comment]
Hello, there are currently no limitations and will not be any until v1.0 which is still at least 6 months away. After that, we may introduce resource restrictions as needed. 

For now, please use PocketHost and share your experience. That is very valuable for the community :)

Let me know if you have any questions.

**Reply by aonghas** [reply]
Reckon you can keep it free till at least end of September? I'm planning on using it for doing some fun things for a wedding (guest photo uploads, chat, etc).

**Reply by benallfree** [reply]
Definitely @aonghas
